ext/mysql2/mysql2_ext.h

```diff
@@ -20,6 +20,9 @@ void Init_mysql2(void);
 #include <mysql/errmsg.h>
 #include <mysql/mysqld_error.h>
 #endif
+#ifdef HAVE_MYSQL_MYSQL_VERSION_H
+#include <mysql/mysql_version.h>
+#endif
```

Comment:
At the risk of a too many combinations, should this also be conditional on the mysql/ prefix as above?